### ISSUE TITLE
Some details about the dataset.

### ISSUE THREAD

**Issue by Carol-gutianle** [yihuaihong/ConceptVectors#2]
Hello, I encountered two questions when I tried to do experiments with `ConceptVectors`.
1. What is the difference between concepts w/wo the key "id"?
![image](https://github.com/user-attachments/assets/655773f9-c2f4-4eb3-baa4-0d61362badb0)
2. How to cope with the same concept with two different (Layer, Dim)?
![image](https://github.com/user-attachments/assets/531b4687-aaea-4bb3-b8e9-383c75a14f5f)
![image](https://github.com/user-attachments/assets/8d646006-1b58-41dc-9661-4f38b5de5b4e)
Thank you for your time!

**Comment by yihuaihong** [comment]
Hi, Tianle.

Please ignore the "id" key. All sample data can be treated equally. (it's likely just an oversight in data processing and filtering. so some samples dont have a "id", I will fix them later). The most crucial distinguishing criterion between samples is their **location**, which is composed of "Layer" and "Dim".

Each sample in ConceptVectors is an independent sub-experiment, regardless of whether the concept it relates to is the same. So e.g. if you want to do a experiment on the "Astrological sign", you can just use each of them to perform the experiment, like doing Needle only on the Layer26. Dim878.  The final performance results in the main table in the paper is the average of every sample's result.

Please feel free to let me know if there are some other questions.

**Comment by Carol-gutianle** [comment]
Thank you for your response. I have one more question. When I tried to reproduce the results of Gradient Ascent on a specific concept, I found that training for just one epoch led to catastrophic collapse of the model. Could you please share the training parameters used for the baseline in the paper?
![image](https://github.com/user-attachments/assets/3ab28bcb-69c8-466d-97da-729985036049)

**Comment by yihuaihong** [comment]
Gradient Ascent is a very unstable baseline, because it didn't control the gradient optimization direction well. Please use the learning rate at the scale of e-6 or even e-7 for this baseline.

If you just would like to see a stable a reasonable performance, I would recommend you use the NPO. You can easily set up the hypeparameters as described in README and reproduce the results for this one.

**Comment by Carol-gutianle** [comment]
Thank you so much for your response—it was really helpful! Best of luck with your research.

**Comment by yihuaihong** [comment]
Thank you as well for your interest!👍
